networking-calico/networking_calico/plugins/ml2/drivers/calico/test/test_plugin_etcd.py

```diff
@@ -892,8 +905,31 @@ def test_start_two_ports(self):
         self.assertEtcdWrites(expected_writes)
         self.assertEtcdDeletes(set())
 
+        # Change network used
+        _log.info("Change network used by endpoint HELLO")
+        context._port['network_id'] = 'calico-other-network-id'
+        self.osdb_ports[0]['network_id'] = 'calico-other-network-id'
+        self.simulated_time_advance(mech_calico.RESYNC_INTERVAL_SECS)
+        self.driver.update_port_postcommit(context)
+        ep_hello_value_v3['metadata']['labels'][
+            'projectcalico.org/openstack-network-name'] = 'my-first-network'
+        ep_hello_value_v3['metadata']['annotations'][
+            'openstack.projectcalico.org/network-id'] = \
+            'calico-other-network-id'
+        self.assertEtcdWrites({ep_hello_key_v3: ep_hello_value_v3})
+        self.assertEtcdDeletes(set())
```

Comment:
@nelljerram I haven't gotten this to work yet, my closest attempts so far being:
1. Use the `osdb_ports` change and `simulated_time_advance`.  This properly updates the `network-name` and `network-id` but leaves behind the `uid` and `creationTimestamp`
2. Use the `osdb_ports` change, `simulated_time_advance` and `context._port` + `update_port_postcommit`. This deletes the `uid` and `creationTimestamp` as expected, and makes the proper changes to the labels but writes down the security group as well. 
  
Not sure what the solution in this case is, could use some insight here.